### PR TITLE
Fix formatting bugs and missing link in termsOfService.mdx

### PR DESCRIPTION
**Pull request by jeff2d2**
Fixes some small formatting bugs and a missing link. These used to be present and lost their formatting somehow when the ToS was moved from vets-api-clients to developer-portal. [See commit to remove the ToS from vets-api-clients for what it used to and should be](https://github.com/department-of-veterans-affairs/vets-api-clients/commit/08613b2e5bba9e0b10748a83553580ffa0174f04#diff-27f7aa4823cb1bb9539b8f99be5ab7ab).

### CODE REVIEW

File: src/content/termsOfService.mdx [hunk 1]
```diff
@@ -53,7 +55,4 @@ VA’s failure to exercise or enforce any right or provision of this Agreement s
...
 
-You further acknowledge that when records regarding an individual are obtained through a VA API, you may not expose that content to other individuals or third parties without specific, explicit consent from the individual or his or her authorized representative, or as permitted by applicable law. The terms “individual” and “record” have the meanings given in the Privacy Act at 5 U.S.C. § 552a(a). If you would like more information about the application of the Privacy Policy at VA, click here.
-
-
-
+You further acknowledge that when records regarding an individual are obtained through a VA API, you may not expose that content to other individuals or third parties without specific, explicit consent from the individual or his or her authorized representative, or as permitted by applicable law. The terms “individual” and “record” have the meanings given in the Privacy Act at 5 U.S.C. § 552a(a). If you would like more information about the application of the Privacy Policy at VA, [click here](https://www.va.gov/privacy/).
```
Comment (rtravitz) [inline src/content/termsOfService.mdx]: What changed in this line? It's hard to tell subtle changes in git.

File: src/content/termsOfService.mdx [hunk 2]
```diff
@@ -2,7 +2,8 @@
 Thank you for using U.S. Department of Veterans Affairs (VA) APIs. By accessing or using our APIs, you are agreeing to the terms below. 
 
 Please review these terms carefully. Once accepted, these terms become a binding legal commitment between you and the VA. If you do not agree to be bound by these terms, you should not use VA’s APIs. 
-Scope
+
+#### Scope
```
Comment (rtravitz) [inline src/content/termsOfService.mdx]: It looks like you need to change this to an H3 (just drop one of the `#`) based on the accessibility error in Jenkins.

Comment (nfasulo) [reply src/content/termsOfService.mdx]: I think it has to be an H2.

File: src/content/termsOfService.mdx (hunk 1 above)
Comment (jeff2d2) [reply src/content/termsOfService.mdx]: At the end: "click here." -> "[click here](https://www.va.gov/privacy/). Then it looks like it picked up deletion of a couple blanks lines.

File: src/content/termsOfService.mdx (hunk 2 above)
Comment (jeff2d2) [reply src/content/termsOfService.mdx]: Yep H2. Tried 3 and that errored also.

**Pull request merged**